requirements.txt

```diff
@@ -0,0 +1,63 @@
+asttokens==2.4.1 ; python_version >= "3.10" and python_version < "4.0"
```

Comment:
is this needed? should it be added to gitignore?

Reply:
I think it's fine to ignore, will test locally and remove if nothing breaks.

Reply:
Doesn't look like we need to export, so removed in https://github.com/canonical/data-platform-helpers/pull/18/commits/80bb74bc28604b49f95e925cc76e40dadf4dccd0